apps/frontend/src/components/cards/EvaluationInfo.vue

```diff
@@ -63,6 +85,21 @@ export default class EvaluationInfo extends Vue {
     return EvaluationModule.evaluationForFile(this.file_object);
   }
 
+  convertGroupsToIVuetifyItems(groups: IGroup[]): IVuetifyItems[] {
+    return groups.map((group) => {
+      return {
+        text: group.name,
+        value: group.id
+      }
+    })
+  }
+
+  updateGroupsForEvaluation(): void {
+    if(this.evaluation?.id){
+      axios.get<IGroup[]>(`/evaluations/${this.evaluation?.id}/groups`).then(({data}) => this.groups = this.convertGroupsToIVuetifyItems(data));
+    }
+  }
```

Comment:
![Screen Shot 2021-06-09 at 9 29 46 AM](https://user-images.githubusercontent.com/2308869/121363776-3d931180-c905-11eb-8447-943359e15f2c.png)

This ends up causing the request to be duplicated for every page load since very similar code is in the EditEvaluationModal. As groups are added to more locations it may make sense to get ahead of this and store groups in the vuex store on evaluations as well.

Looking back on the original implemention of this, it seems that it would make more sense to just return the group as part of the `EvaluationDto`. With these changes we are requiring the groups in order to group the data by evaluations anyway, we may as well just include it all the time and avoid the need for this call altogether, leveraging the `evaluations.ts` store instead.